.changeset/famous-lemons-sneeze.md

```diff
@@ -0,0 +1,22 @@
+---
+"@comet/cms-admin": minor
+---
+
+Add `tags` property to block factories to support adding translatable key words and improve searching for blocks in the `AddBlockDrawer`
+
+Example usage:
+
+```tsx
+createCompositeBlock({
+    name: "Media",
+    displayName: "Media Block",
+    blocks: {
+        /* ... */
+    },
+    tags: [
+        defineMessage({ id: "standaloneMedia.tag.image", defaultMessage: "Image" }),
+        defineMessage({ id: "standaloneMedia.tag.video", defaultMessage: "Video" }),
```

Comment:
the example is a bit outdated, as image and video are now automatically inherited